adapter/persistence/src/main/java/org/anastaklo/portal/adapter/persistence/BaseJpaEntity.java

```diff
@@ -0,0 +1,34 @@
+package org.anastaklo.portal.adapter.persistence;
+
+import java.util.Objects;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+public class BaseJpaEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Override
+    public int hashCode() {
+
+        return 13;
```

Comment:
what the magic?)

Reply:
I borrowed this idea from this article https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/, see section `Fixing the entity identifier equals and hashCode`

> there is only one solution: the hashCode should always return the same value